packages/waku/src/lib/utils/merge-vite-config.ts

```diff
@@ -29,9 +29,12 @@ export async function mergeUserViteConfig(config: UserConfig) {
       // FIXME weird error around plugin duplication when removed
```

Comment:
I'm not confident with this approach at all. I hope someone with more Vite experience would help us. 🙏